hnn_core/network.py

```diff
@@ -216,7 +239,103 @@ def __repr__(self):
                  len(self.pos_dict['L5_basket'])))
         return '<%s | %s>' % (class_name, s)
 
-    def _add_external_feeds(self):
+    def attach_drive(self, drive_name, drive_params=None,
```

Comment:
is there a conceptual difference between "drives" and "feed"? I'd suggest sticking to the historical name for now and revamping the name through the entire repo in a separate PR if you feel one should be preferred over the other

Reply:
I think the difference relates to causality? A drive is likely something produced by an event (tactile stimulus, etc.) whereas a feed is more generic activity (baseline/noise). Not sure there is a consensus in literature on this terminology though...